test/context_test.rb

```diff
@@ -111,6 +112,24 @@ def test_unsupported_api_version
       end
     end
 
+    def test_host_scheme
+      ShopifyAPI::Context.setup(
+        api_key: "key",
+        api_secret_key: "secret",
+        api_version: "unstable",
+        host_name: "host",
+        scope: ["scope1", "scope2"],
+        is_private: true,
+        is_embedded: true,
+        session_storage: ShopifyAPI::Auth::FileSessionStorage.new,
+        is_https: false,
```

Comment:
That's true, when using the CLI we'll be fetching these 2 values from the `HOST` env var, so it'd be cleaner to just parse the scheme we get itself rather than checking it for a boolean.

Also, because this came back to bite us on the PHP template - will `URI().host` include the port in an address like `localhost:1234`?

Reply:
Ya, I went back and forth on this one. I initially had it as `hots_scheme` as you've outlined, but then I had to think about validation logic for making sure the provided value was valid (was one of `http://` or `https://`) and thought the boolean config was easier. The drift is a pain though..

I do like the idea of parsing from `HOST`, as that is fewer things to configure / validate. Can we assume that env variable will always be there? I'm not seeing that in the readme for the API like it is for `shopify_app`.